src/components/menu/menu-gestures.ts

```diff
@@ -57,14 +57,14 @@ export class MenuContentGesture extends SlideEdgeGesture {
   }
 
   onSlide(slide: SlideData, ev: any) {
-    const z = (this.menu.isRightSide ? slide.min : slide.max);
+    const z = ((this.menu.isRightSide && !this.plt.isRTL) || (!this.menu.isRightSide && this.plt.isRTL) ? slide.min : slide.max);
```

Comment:
well technically yes but I put it that way for more readability ...
Ok I will change all of em